### PR TITLE
Feature/forwardheader refactor

### PR DESCRIPTION
**Pull request by acn-dgopa**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj [hunk 1]
```diff
@@ -3,14 +3,14 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <OutputType>Library</OutputType>
-    <AssemblyVersion>3.0.0.0</AssemblyVersion>
-    <FileVersion>3.0.0.0</FileVersion>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```
Comment (Alxandr) [inline src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj:6]: These two fields shouldn't bee necessary, no? If you remove them, don't you get the correct assembly version and file version?

**Review by Alxandr — APPROVED** [review]

File: src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj (hunk 1 above)
Comment (acn-dgopa) [reply src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj:6]: haven't tried it without them. Can check it out

Comment (acn-dgopa) [reply src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj:6]: Yes, if not set, it defaults to 1.0.0

Comment (Alxandr) [reply src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj:6]: I don't think this is correct, as I get the following output from msbuild:

image

Comment (Alxandr) [reply src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj:6]: Ah, it's cause you haven't set Version. You should replace <PackageVersion> with just <Version>. It sets all 3.

**Pull request merged**